lib/routes/denonbu/news.ts

```diff
@@ -0,0 +1,187 @@
+        },
+        {
+            source: ['denonbu.jp/movie'],
+            target: '/news/movie',
+        },
+        {
+            source: ['denonbu.jp/music'],
+            target: '/news/music',
+        },
+        {
+            source: ['denonbu.jp/livearchives'],
+            target: '/news/livearchives',
+        },
+
+        {
+            source: ['denonbu.jp/area/:area'],
+            target: '/news/:area',
+        },
+    ],
+    name: '新闻',
+    maintainers: ['outloudvi'],
+    handler,
+};
+
+const AREA_NAMES = {
+    akiba: '外神田文芸高校',
+    harajuku: '神宮前参道學園',
+    azabu: '港白金女学院',
+    shibuya: '帝音国際学院',
+    kabuki: '真新宿GR学園',
+    'deep-okubo': 'Bellemule（深大久保DJ＆ダンスアカデミー）',
+    'deep-okubo-k': '輝きノスタルジア（深大久保DJ＆ダンスアカデミー）',
+    shinsaibashi: "OKINI☆PARTY'S（心斎橋演芸高校）",
+    ikebukuro: '池袋電音部（池袋空乗院高校）',
+    neotokyo: '東京電脳（東京電脳学園）',
+    neonakano: '中野電脳（中野電脳学園）',
+    shimokitazawa: "Ma'Scar'Piece（北沢音箱高校）",
+
+    news: '新闻',
+    event: '活动',
+    goods: '商品',
+    comic: '漫画',
+    movie: '影片',
+    music: '音乐',
+    livearchives: 'Live留档',
+};
+
+const BASE_URL = 'https://denonbu.jp/backend-api/v1.0.0/';
+const COMMON_HEADERS = {
+    'X-API-KEY': 'FVpHcMLqyf7v2EubqiLxznC9gVMqBDFFMt4zvkS2',
+};
+const PRIMARY_CATEGORIES = new Set(['news', 'event', 'goods', 'comic', 'movie', 'music', 'livearchives']);
+
+function getToken(): Promise<string> {
+    return ofetch(String(new URL('auths/token/get', BASE_URL)), {
+        headers: COMMON_HEADERS,
+    }).then((x) => x.payload.token);
+}
+
+function buildLink(body: any): string | null {
+    switch (body.source_type) {
+        case 'main':
+        case 'deep-okubo':
+        case 'shinsaibashi':
+        case 'neotokyo': {
+            const { sid, uid } = body;
+            if (sid !== undefined && uid !== undefined) {
+                return `https://denonbu.jp/detail/${sid}/${uid}`;
+            }
+            return null;
+        }
+        case 'tw': {
+            const {
+                account: { account_id },
+                uid,
+            } = body;
+            if (account_id && uid !== undefined) {
+                return `https://twitter.com/${account_id}/status/${uid}`;
+            }
+            return null;
+        }
+        default:
+            return null;
+    }
+}
+
+async function handler(ctx: Context): Promise<Data> {
+    const { area: _area } = ctx.req.param();
+    const area = _area ?? 'news';
+    const token = await getToken();
+    const data = await ofetch(String(new URL(`contents/search/${area}?limit=20&offset=0`, BASE_URL)), {
+        headers: {
+            ...COMMON_HEADERS,
+            Authorization: `Bearer ${token}`,
+        },
+    }).then((x) => x.payload.items);
+
+    const items = data.map((item) => {
+        const { title, body, post_date, category, media } = item;
+        const link = buildLink(item);
+        const result: DataItem = {
+            title: title ?? body.split('\n')[0] + '...',
```

Comment:
```suggestion
            title: title ?? body.split('\n')[0],
```
This conflicts with https://github.com/DIYgod/RSSHub/blob/71837553ca09dee8aa591be16617be0e7644edd2/lib/middleware/template.tsx#L50

Reply:
[6273fc0](https://github.com/DIYgod/RSSHub/pull/20770/commits/6273fc03392125de7e14516ac0123c61fd0448a1)